src/styles/main.scss

```diff
@@ -19,6 +19,8 @@
 
 html {
   @include font-settings;
+  // Including NYPL DS gave the footer extra space at the bottom.
```

Comment:
@scooooooooby do we have a ticket to fix this?

Reply:
No, so I'll add one.